<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>441</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>二维码处理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>670</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>二维码</string>
     </attribute>
     <widget class="QLineEdit" name="file_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_savePic">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buildCode_Btn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_openFile">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string>显示区域</string>
      </property>
      <widget class="QLabel" name="image">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>231</width>
         <height>231</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>70</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string>生成区域</string>
      </property>
      <widget class="qr_widget" name="QR_widget" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>20</y>
         <width>231</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="chooseCode_Box">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>QR码</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DM码</string>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="text_input">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>350</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Recog_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>条形码</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>641</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string>生成区域</string>
      </property>
      <widget class="qr_widget" name="QR_widget_bar" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>621</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string>显示区域</string>
      </property>
      <widget class="QLabel" name="image_bar">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>621</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="chooseCode_Box_bar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>350</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>CODE11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CODE39</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CODE128</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="buildCode_Btn_bar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="text_input_bar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_savePic_bar">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_openFile_bar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file_name_bar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Recog_text_bar">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qr_widget</class>
   <extends>QWidget</extends>
   <header>qr_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
